<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndDirectionFocusing</class>
 <widget class="QDialog" name="wndDirectionFocusing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Farfield - Beam focusing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QLabel" name="lblFreqMax">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblZenMax">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lblAzMin">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblZen">
     <property name="text">
      <string>Zenith</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QSlider" name="sldFrequency">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblZenMin">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblFreq">
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblFreqMin">
     <property name="text">
      <string>7GHz</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblAz">
     <property name="text">
      <string>Azimuth</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QSlider" name="sldZenith">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSlider" name="sldAzimuth">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="lblAzMax">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="4" colspan="4">
    <widget class="QVTKOpenGLNativeWidget" name="plot_antenna_diagram" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
